flowauth/backend/flowauth/token_management.py

```diff
@@ -226,3 +223,55 @@ def add_token(server):
     db.session.add(token)
     db.session.commit()
     return jsonify({"token": token_string, "id": token.id})
+
+
+def generate_token(
```

Comment:
Given that we _are_ duplicating the `generate_token()` function, can we add a comment here and also in `flowkit_jwt_generator` that a duplicate version exists and that they should be changed in parallel?

Reply:
I've added a test that covers the duped generator, but fine to add a comment as well.

Reply:
Yes, great to have the test. 👍Just figured it's useful to have a comment too so that you don't have to wait for a failing CI run to discover you should have changed the other implementation, too.